### ISSUE TITLE
[BUG] Inventory skew logic in PMM strategy should use total balance rather than available balance to calculate order size

### ISSUE THREAD

**Issue by martinkou** [CoinAlpha/hummingbot#1927]
**Describe the bug**
The beginning of the `c_apply_inventory_skew()` function currently reads like this...

```
    cdef c_apply_inventory_skew(self, object proposal):
        cdef:
            MarketBase market = self._market_info.market
            object base_balance = market.c_get_available_balance(self.base_asset)
            object quote_balance = market.c_get_available_balance(self.quote_asset)
```

It is using available balance for adjust order sizes for inventory skew. This is incorrect and is a regression from previous behavior - inventory skew is supposed to keep the overall balance within a fixed ratio range, not the available balance.

**Release version**
0.28.0

**Comment by martinkou** [comment]
Upon further investigation, we've been doing this before v0.28.0 and thus it's not a regression.

The bug reported by Reuben from ZCoin is actually caused by a separate issue - the behavior of the trading algorithm was technical correct, but the status message reported a different behavior. The user was instead misled by the wrong status message about what his inventory skew parameters would do.

**Comment by dennisocana** [comment]
Fixed in version 0.31
